src/main.js

```diff
@@ -2,11 +2,48 @@ import Vue from "vue";
 import App from "./App.vue";
 import router from "./router";
 import store from "./store";
+import vuetify from "./plugins/vuetify";
+import axios from "axios";
+import { ValidationProvider, setInteractionMode } from "vee-validate";
+import "./components/validationRules";
+import authService from "./services/authService";
+
+Vue.prototype.$authService = authService;
+
+Vue.component("ValidationProvider", ValidationProvider);
+setInteractionMode("eager");
 
 Vue.config.productionTip = false;
+axios.defaults.baseURL = "/api";
 
 new Vue({
   router,
   store,
+  vuetify,
+  created() {
+    const userDetails = localStorage.getItem("user");
+    if (userDetails) {
+      const userData = JSON.parse(userDetails);
+      this.$store.commit("setUserData", userData);
+    }
+    axios.interceptors.response.use(
+      response => response,
+      error => {
+        return Promise.reject(error);
+      }
+    );
```

Comment:
There is useful place to handle 401. If you will get it here, logout user instantly